src/main/scala/codacy/tailor/Tailor.scala

```diff
@@ -35,69 +30,95 @@ object Tailor extends Tool {
 
   private lazy val configFileNames = Set(".tailor.yml")
 
-  override def apply(source: Source.Directory,
-                     configuration: Option[List[Pattern.Definition]],
-                     files: Option[Set[Source.File]],
-                     options: Map[Options.Key, Options.Value])(
-      implicit specification: Specification): Try[List[Result]] = {
-    Try {
+  private def getNativeConfigurationFromSource(source: Source.Directory): Option[List[String]] = {
+    configFileNames
+      .map(name => Try(File(source.path) / name))
+      .collectFirst {
+        case Success(file) if file.isRegularFile =>
+          List("-c", file.toJava.getAbsolutePath)
+      }
+  }
 
-      lazy val nativeConfig = configFileNames
-        .map(name => Try(File(source.path) / name))
-        .collectFirst {
-          case Success(file) if file.isRegularFile =>
-            List("-c", file.toJava.getAbsolutePath)
-        }
+  private def getPatternsToUseFromCodacyConfig(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [getPatternsToUseFromCodacyConfig should have a smaller scope by using private[this]](https://app.codacy.com/gh/codacy/codacy-tailor/pullRequest?prid=4816796)